# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/xul/XULMenuAccessible.cpp
# Commit: c0a7fdc4bfca
# Full Hash: c0a7fdc4bfca0596ffecd347f0e248440e371989
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2022-01-05 09:33:53
# Regressor Bug: 1737944
# File Overlap Count: 2
# Description:
#   Bug 1737944 - P5: Unify GetLevelInternal and GetDefaultLevel as Accessible::GetLevel. r=Jamie
#   
#   The nsAccUtils method was a "fast" one for calculating set sizes and
#   conceptual parents. Unified it with
#   LocalAccessible::GetLevelInternal in Accessible::GetLevel with an
# ==============================================================================

diff -r f6ce0e05c92e -r c0a7fdc4bfca accessible/xul/XULMenuAccessible.cpp
--- a/accessible/xul/XULMenuAccessible.cpp	Tue Jan 04 21:01:35 2022 +0000
+++ b/accessible/xul/XULMenuAccessible.cpp	Tue Jan 04 21:01:36 2022 +0000
@@ -240,7 +240,7 @@
   return roles::MENUITEM;
 }
 
-int32_t XULMenuitemAccessible::GetLevelInternal() {
+int32_t XULMenuitemAccessible::GetLevel(bool aFast) const {
   return nsAccUtils::GetLevelForXULContainerItem(mContent);
 }
 